ext/gd/gd.stub.php

```diff
@@ -199,11 +199,11 @@ function imagecolortransparent(GdImage $im, int $col = UNKNOWN): ?int {}
 
 function imageinterlace(GdImage $im, int $interlace = UNKNOWN): ?int {}
 
-function imagepolygon(GdImage $im, array $points, int $num_pos, int $col): bool {}
+function imagepolygon(GdImage $im, array $points, int $num_points, int $col = UNKNOWN): bool {}
```

Comment:
```suggestion
function imagepolygon(GdImage $im, array $points, int $num_points_or_col, int $col = UNKNOWN): bool {}
```
Maybe?

Reply:
That's definitely more correct for reflection, but might be confusing wrt. the deprecation warning ("the $num_points parameter is deprecated"). However, having an optional parameter in the middle of the parameter list is confusing anyway, but as it would only be temporary, guess it's okay.